app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaScreenModel.kt

```diff
@@ -734,15 +736,19 @@ class MangaScreenModel(
                 chapters = chapters.toTypedArray(),
             )
 
-            if (!read) return@launchIO
+            if (!read ||
+                successState?.hasLoggedInTrackers == false ||
+                autoTrackState == AutoTrackState.NEVER
+            ) {
+                return@launchIO
+            }
 
             val tracks = getTracks.await(mangaId)
             val maxChapterNumber = chapters.maxOf { it.chapterNumber }
             val shouldPromptTrackingUpdate = tracks.any { track -> maxChapterNumber > track.lastChapterRead }
 
             if (!shouldPromptTrackingUpdate) return@launchIO
-
-            if (trackPreferences.autoUpdateTrackOnMarkRead().get()) {
+            if (autoTrackState == AutoTrackState.ALWAYS) {
```

Comment:
You haven't used the ASK option?
Should it be showing the toast to confirm when settings is ASK?

Reply:
Well if it's anything but ASK it would have hit a return statement. So what the point?
![Screenshot_20241025_230506_Chrome](https://github.com/user-attachments/assets/9d9690f2-3b69-4496-985a-ff5dac5fad17)

Reply:
A toast feels unnecessary for ask cus you are already pressing okay on the snackbar

Reply:
> A toast feels unnecessary for ask cus you are already pressing okay on the snackbar

the OK snackbar is what I'm talking about. But you are right, a return is sufficient.